tests/cases/fourslash/commentsLinePreservation.ts

```diff
@@ -129,22 +129,22 @@ verify.quickInfos({
     3: ["(parameter) param1: string", "first line of param\n\nparam information third line"],
 
     g: ["function g(param1: string): void", "This is firstLine\nThis is second Line"],
-    4: ["(parameter) param1: string", " param information first line"],

```

Comment:
I like the old output better; `@param` is supposed to set a margin so that the space is preserved in `param information first line`. Preserving the newlines too is OK, depending on how it looks when formatted in VS Code.

This is true of all the changes below.

Reply:
current VSCode version
![image](https://user-images.githubusercontent.com/1636026/162528022-47933c74-0d0b-471e-9c59-ab53662f321b.png)
![image](https://user-images.githubusercontent.com/1636026/162528043-2a56adc5-0ac1-4fa0-b430-c43ef75310aa.png)
![image](https://user-images.githubusercontent.com/1636026/162528067-2dfd124f-c468-482b-8e6e-a701ba318209.png)

VS Code with my changes
![image](https://user-images.githubusercontent.com/1636026/162527884-7ac5f985-1b96-43f3-8419-5d05c3a50588.png)
![image](https://user-images.githubusercontent.com/1636026/162527854-6b9e3275-7a59-45b5-8a29-1ec61ce73aa3.png)
![image](https://user-images.githubusercontent.com/1636026/162527826-7d4fd64d-26e6-4889-811c-127a1a764e8f.png)

For the the later looks better 😄